### PR TITLE
feat: forgot password page

### PR DESCRIPTION
**Pull request by suviz**
## WHY:
Advances [#189](https://github.com/OpenVAA/voting-advice-application/issues/189)

### What has been changed (if possible, add screenshots, gifs, etc. )
Implements the forgot password page where users can input their email address in order to receive a password reset link by email.

## Check off each of the following tasks as they are completed

- [x] I have reviewed the changes myself in this PR. Please check the [self-review document](https://github.com/OpenVAA/voting-advice-application/blob/main/docs/contributing/self-review.md)
- [ ] I have added or edited unit tests.
- [ ] I have run the unit tests successfully.
- [x] I have tested this change on my own device.
- [ ] I have tested this change on other devices (Using Browserstack is recommended).
- [x] I have tested my changes using the [WAVE extension](https://wave.webaim.org/extension/)
- [x] I have added documentation where necessary.
- [x] Is there an existing issue linked to this PR?

**Clean up your git commit history before submitting the pull request!**

### CODE REVIEW

**Review by TrixterTheTux — APPROVED** [review]
LGTM

File: frontend/src/lib/components/authentication/LoginPage.svelte [hunk 1]
```diff
@@ -73,6 +73,8 @@ Candidate login page. This component also takes care of the login process.
           {/if}
           <button type="submit" class="btn btn-primary mb-md w-full max-w-md"
```
Comment (norppaa) [inline frontend/src/lib/components/authentication/LoginPage.svelte:74]: Should use the new button component

**Review by norppaa — APPROVED** [review]
Looks good to me!

File: frontend/src/lib/components/authentication/LoginPage.svelte (hunk 1 above)
Comment (suviz) [reply frontend/src/lib/components/authentication/LoginPage.svelte:74]: I think it's better to keep to the old style and not use the component yet, in order to not mix styles. This file should be refactored later to use the new components.

**Review by joonatan-aatos — CHANGES_REQUESTED** [review]

**Pull request merged**